generators/typescript/sdk/versions.yml

```diff
@@ -1,4 +1,19 @@
 # yaml-language-server: $schema=../../../fern-versions-yml.schema.json
+- version: 2.6.0
+  changelogEntry:
+    - summary: |
+        Users can now pass in `queryParams` as part of the request options.
+        ```ts
+        await client.foo.bar(..., {
+          queryParams: {
```

Comment:
nit: i would do `queryParameters` not `queryParams` but leave that decision to you! (to align with fetcher)